FiligranHQ/xtm-hub
# [frontend/ci] feat: add version of XTM Hub on UI

# Context: 

This PR adds the version of XTM Hub to a brand new **Parameters** page in the admin part of the application. 
This implementation aligns with the display approach used for OBAS and OCTI, ensuring consistency across the product suite.

The versioning logic follows these rules:
- `0.0.0-dev` is displayed for local dev environments.
- The Git commit SHA1 on which the Docker image is built is displayed for `staging` and `development`.
- The Git tag is displayed for production.

![Capture d’écran 2025-01-08 à 14 36 31](https://github.com/user-attachments/assets/b59b2587-1429-4445-8f83-e68c1106160f)
![Capture d’écran 2025-01-08 à 14 36 38](https://github.com/user-attachments/assets/63169bbe-6041-4e03-ac4e-6bf2a270161c)


# How to test:  

1. Pull the branch `issue/114`.
2. Start the application locally:
   - Verify that the version displayed is `0.0.0-dev`.
3. Deploy to `staging` or `development`:
   - Verify that the displayed version corresponds to the Git commit SHA1.
4. Deploy to `production`:
   - Verify that the displayed version corresponds to the Git tag.

# What tests have been made: 

# Additional information: 

- This implementation dynamically retrieves and displays the version information based on the environment.
- The change ensures alignment with versioning practices across Filigran products.

Close #114
